BikeRentalApi/serializers/stationSerializer.py

```diff
@@ -1,8 +1,20 @@
-from BikeRentalApi.models import BikeStation
+from rest_framework.fields import SerializerMethodField
+
+from BikeRentalApi.enums import BikeState
+from BikeRentalApi.models import BikeStation, Bike
 from rest_framework import serializers
 
 
 class StationSerializer(serializers.ModelSerializer):
+    status = SerializerMethodField()
+    activeBikesCount = SerializerMethodField()
+
     class Meta:
         model = BikeStation
         exclude = ['state']
+
+    def get_status(self, obj: BikeStation):
+        return obj.state.label
+
+    def get_activeBikesCount(self, obj: BikeStation):
+        return Bike.objects.filter(station__pk__exact = obj.pk, bike_state = BikeState.Working).count()
```

Comment:
Can be replaced with `obj.bike_set.all().count()`

Reply:
changed to `obj.bike_set.filter(bike_state = BikeState.Working).count()` to return anly avalible bikes